src/pyfmi/fmil3_import.pxd

```diff
@@ -45,23 +60,55 @@ cdef extern from 'fmilib.h':
         fmi3_xml_element_end_handle_ft   endHandle
         void* context
 
+    ctypedef void(*fmi3_log_message_callback_ft)(
+        fmi3_instance_environment_t instance_environment,
+        fmi3_status_t status,
+        fmi3_string_t category,
+        fmi3_string_t message,
+        )
+
     cdef struct fmi3_import_t:
         pass
+
+
     # FMI SPECIFICATION METHODS (3.0)
     # BASIC
+    int fmi3_import_create_dllfmu(fmi3_import_t*, fmi3_fmu_kind_enu_t, fmi3_instance_environment_t, fmi3_log_message_callback_ft )
+    FMIL.jm_status_enu_t fmi3_import_instantiate_model_exchange(
+        fmi3_import_t* fmu,
+        fmi3_string_t instanceName,
+        fmi3_string_t resourcePath,
+        fmi3_boolean_t visible,
+        fmi3_boolean_t loggingOn
+    )
     # modes
-
+    fmi3_status_t fmi3_import_enter_initialization_mode(
+        fmi3_import_t* fmu,
+        fmi3_boolean_t toleranceDefined,
+        fmi3_float64_t tolerance,
+        fmi3_float64_t startTime,
+        fmi3_boolean_t stopTimeDefined,
+        fmi3_float64_t stopTime)
+    fmi3_status_t fmi3_import_exit_initialization_mode(fmi3_import_t* fmu)
+    fmi3_status_t fmi3_import_enter_event_mode(fmi3_import_t* fmu)
+    fmi3_status_t fmi3_import_enter_continuous_time_mode(fmi3_import_t* fmu)
+    fmi3_status_t fmi3_import_enter_event_mode(fmi3_import_t* fmu)
```

Comment:
```suggestion
```
See 2 lines above, you seem to have added it twice by accident.

Reply:
Fixed in 2b3b66e